api/nakadi-event-bus-api.yaml

```diff
@@ -574,6 +574,25 @@ paths:
             The flow id of the request, which is written into the logs and passed to called services. Helpful
             for operational troubleshooting and log analysis.
           type: string
+        - name: X-nakadi-cursors
+          in: header
+          description: |
+            Cursors indicating the partitions and offsets consumed by the client.
+
+            When this header is provided, Nakadi calculates the consumer lag, e.g. the number of events between the
+            provided offset and the most recent published event.
+
+            In this situation, the response will contain the `unconsumed_events` property for every partition.
+
+            The lag calculation is non inclusive, e.g. if the provided offset is the offset of the latest published
+            event, the number of `unconsumed_events` will be zero.
+
+            If this header is not present, partitions won't display `unconsumed_events`.
```

Comment:
I would say it should be possible to pass only the cursors of interest. What do you think? Is it clear from the description?